ddtrace/contrib/trace_utils.py

```diff
@@ -243,8 +245,26 @@ def set_http_meta(
     request_headers=None,  # type: Optional[Mapping[str, str]]
     response_headers=None,  # type: Optional[Mapping[str, str]]
     retries_remain=None,  # type: Optional[Union[int, str]]
+    raw_uri=None,  # type: Optional[str]
+    request_cookies=None,  # type: Optional[Dict[str, str]]
+    request_path_params=None,  # type: Optional[Dict[str, str]]
 ):
     # type: (...) -> None
+    """
+    Set HTTP metas on the span
+
+    :param method: the HTTP method
+    :param url: the HTTP URL
+    :param status_code: the HTTP status code
+    :param status_msg: the HTTP status message
+    :param query: the HTTP query part of the URI as a string
+    :param request_headers: the HTTP request headers
+    :param response_headers: the HTTP response headers
+    :param raw_uri: the full raw HTTP URI (including ports and query)
+    :param request_cookies: the HTTP request cookies as a dict
+    :param request_path_params: the representation of the framework URL without the request data for instance,
+        /posts/:id for a route where /posts/<id:int> is allowed
```

Comment:
We generally refer to this as the "route". tag/address would be `http.route`.

"path params", sounds like you want `{"id": "1234"}`, but you actually want `/posts/:id` ?

Reply:
good catch, I updated the docstring